AssertionError("termination_mis != MOI.OTHER_ERROR")
